kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerRetrofitErrorHandler.java

```diff
@@ -59,11 +59,11 @@ public Throwable handleError(RetrofitError e) {
         }
         return retval;
       case NETWORK:
-        return new SpinnakerNetworkException(e.getMessage(), e.getCause());
+        return new SpinnakerNetworkException(e);
```

Comment:
This becomes easy to change too

```suggestion
        return new SpinnakerNetworkException(e.getMessage(), e.getCause(), e.getUrl());
```